docs/concepts.md

```diff
@@ -105,4 +105,4 @@ a RESTful API to enable third-party integration.
 
 ## Inferno Architecture Diagram
 
-You can view an [Inferno Architecture Diagram](/download/Inferno_Architecture.pdf) (PDF) which summarizes the components of Inferno Framework and includes a glossary of terms.
+You can view an [Inferno Architecture Diagram (PDF 1.1 MB)](/download/Inferno_Architecture.pdf) which summarizes the components of Inferno Framework and includes a glossary of terms.
```

Comment:
Maybe we should be linking this in a separate page if we reference it more than once across pages. Thoughts?

Reply:
I think it's fine to have the links on both pages as is - it isn't changing frequently, and is trivial to move to a partial if needed.